src/test/java/org/cryptimeleon/math/structures/ExpTests.java

```diff
@@ -59,15 +57,11 @@ private static GroupElementImpl naiveEval(Multiexponentiation multiexp) {
 
     @Test
     public void testExpAlgs() {
-        BilinearGroupImpl bilGroup = new BarretoNaehrigBilinearGroupImpl(60);
-
-        SecureRandom secRand = new SecureRandom();
-        Random rand = new Random(secRand.nextLong());
-        for (int i = 0; i < 10; ++i) {
+        for (int i = 0; i < 4; ++i) {
             GroupElementImpl elem = bilGroup.getG1().getUniformlyRandomNonNeutral();
-            BigInteger exponent = BigInteger.valueOf(rand.nextInt());
-            System.out.println("Chosen element: " + elem);
-            System.out.println("Chosen exponent: " + exponent);
+            BigInteger exponent = RandomGenerator.getRandomNumber(BigInteger.valueOf(Integer.MAX_VALUE));
+            //System.out.println("Chosen element: " + elem);
```

Comment:
The printing here was supposed to make the test reproducible in case of an error due to the random choice.

Reply:
Ah! I see. I just commented them out because they were polluting the test output. They're still there to be uncommented in case this becomes relevant again :)